index.html

```diff
@@ -386,8 +386,7 @@
   <body>
     <section id="abstract">
       <p>
-        This document describes a formal information model and a common representation for a Web of Things (WoT) Thing
-        Description 2.0 Version. A Thing Description describes the metadata and interfaces of <a>Things</a>, where a
+        This document describes a formal information model and a common representation for a Web of Things (WoT) Thing Description (TD), Version 2.0. A Thing Description describes the metadata and interfaces of <a>Things</a>, where a
```

Comment:
just to (I hope) make GitHub properly show the changes made by this PR
```suggestion
        This document describes a formal information model and a common representation for a Web of Things (WoT) Thing
        Description (TD), Version 2.0. A Thing Description describes the metadata and interfaces of <a>Things</a>, where a
```